### PR TITLE
Added /document input log and duplicated maxRequestSize size

### PR DESCRIPTION
**Pull request by angelmp01**
- Added info log to print input in /document endpoint
- Increased maxRequestSize from 100m to 200m

### CODE REVIEW

**Review by metmajer — APPROVED** [review]
👍

**Comment by metmajer** [comment]
@angelmp01 the PR points to master, but the changelog lists the changes under v3.0. We need this fix in master, 3.0, and a new branch 4.0.1 (created from master).

**Comment by metmajer** [comment]
FYI @jorge-romero @s2oBCN please support the review

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,16 +6,18 @@
 ### Fixed
 - Github template tests fail in proxy environment ([#56](https://github.com/opendevstack/ods-document-generation-svc/issues/56))
 
-## [3.0] - 2020-08-11
```
Comment (jorge-romero) [inline CHANGELOG.md:9]: Have a look at the changes in the changelog of the ods-jenkins-shared library if we release this with the 4.x it should have a section

Comment (angelmp01) [reply CHANGELOG.md:9]: but I need to fix the CHANGELOG in this PR or the next PR against the 4.x branch?

**Review by s2oBCN — APPROVED** [review]

**Pull request merged**